docs/guides/getting-started-java-spring.md

```diff
@@ -177,6 +179,8 @@ To check your work, visit our [sample repository](https://github.com/camunda/cam
 
 In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
 
+Since the application is now running a worker subscribed to the workflow engine, it will not exit until you hit `Ctrl+C`.
```

Comment:
There's no such thing as obvious, someone will surely be surprised that their app is no longer exiting on its own.

Reply:
Good add. 

Non-blocking feedback: Do we need to add something here for the Mac users? I can talk myself both in and out of it.